src/Service/OS2Forms/CaseSubmissionNormalizers/ResidentAndRentBoardSubmissionNormalizer.php

```diff
@@ -2,129 +2,103 @@
 
 namespace App\Service\OS2Forms\CaseSubmissionNormalizers;
 
-use App\Exception\WebformSubmissionException;
-
-class ResidentAndRentBoardSubmissionNormalizer implements SubmissionNormalizerInterface
+class ResidentAndRentBoardSubmissionNormalizer extends AbstractNormalizer implements SubmissionNormalizerInterface
 {
     /**
      * Handles the shared properties between ResidentComplaintBoardCase and RentBoardCase.
      */
     public function normalizeSubmissionData(string $sender, array $submissionData): array
```

Comment:
Can and should be moved to parent class.

Reply:
Makes sense. Fixed.